py_swf/clients/decision.py

```diff
@@ -157,7 +157,8 @@ def walk_execution_history(
 
             kwargs['nextPageToken'] = next_page_token
 
-    def finish_decision_with_activity(self, task_token, activity_id, activity_name, activity_version, activity_input):
+    def finish_decision_with_activity(self, task_token, activity_id, activity_name, activity_version, activity_input,
+                                      override_config_dict=None):
```

Comment:
If we opt to be able to pass in a dictionary, let's name the parameter something that makes it clear these are _only_ used for timeout overrides. What's the downside of pushing these parameters as arguments (with default of None) to this function?